boxoffice/models/line_item.py

```diff
@@ -255,10 +256,30 @@ def sales_by_date(dates, user_tz, item_ids):
     return date_sales
 
 
-def sales_delta(user_tz, item_ids):
+def calculate_weekly_sales(item_collection_ids, user_tz, year):
```

Comment:
Since, this method runs a query against `line_item` and processes the retrieved data, it made sense to keep it inside the model, along with methods such as `sales_by_date` and `sales_delta`.